src/screens/Habitoptions.js

```diff
@@ -1,132 +1,145 @@
 import React, { useState } from 'react';
-import { View, Text, StyleSheet, TouchableOpacity, FlatList, TextInput, Button } from 'react-native';
+import { View, Text, StyleSheet, TouchableOpacity, ScrollView, TextInput, Button } from 'react-native';
+
+const habitCategories = {
+  'Health and Fitness': ['Exercise', 'Morning Stretching', 'Walk', 'Eat Healthy', 'Cycle', 'Swim', 'Work Out', 'Brush Your Teeth', ],
+  'Personal Growth': ['Read', 'Write in Journal', 'Meditate', 'Limited Screen Time'],
+  'Productivity': ['Complete Daily To-Do List', 'Study'],
+  'Social and Family': ['Spend Time with Family', 'Connect with Friends', 'Express Gratitude'],
+  'Leisure': ['Cooking', 'Outdoor Time', 'Watch a Movie'],
+};
```

Comment:
don't we have the option to add/change these? will they always be the same?

Reply:
There will be some more additional to these and user can already add a new one if he/she wants.